Makefile

```
@@ -99,6 +102,9 @@ MIPS_VERSION := -mips2
 # we support Microsoft extensions such as anonymous structs, which the compiler does support but warns for their usage. Surpress the warnings with -woff.
 CFLAGS += -G 0 -non_shared -Xfullwarn -Xcpluscomm $(IINC) -nostdinc -Wab,-r4300_mul -woff 624,649,838,712
 
+# Use relocations and abi fpr names in the dump
+OBJDUMP_FLAGS := -d -r -Mreg-names=32
```

```suggestion
OBJDUMP_FLAGS := -drz -Mreg-names=32
```
Might as well disassemble zeros as well for full clarity?

---

Good idea. Interestingly, I tried combining the flags like this and it threw an error.